pkg/cloud/packet/client.go

```diff
@@ -138,25 +140,33 @@ func (p *Client) NewDevice(ctx context.Context, req CreateDeviceRequest) (*packn
 
 	userData = stringWriter.String()
 
-	// Allow to override the facility for each PacketMachineTemplate
-	facility := req.MachineScope.PacketCluster.Spec.Facility
-	if req.MachineScope.PacketMachine.Spec.Facility != "" {
-		facility = req.MachineScope.PacketMachine.Spec.Facility
+	// If Metro or Facility are specified at the Machine level, we ignore the
+	// values set at the Cluster level
+	facility := packetClusterSpec.Facility
+	metro := packetClusterSpec.Metro
+
+	if packetMachineSpec.Facility != "" || packetMachineSpec.Metro != "" {
+		metro = packetMachineSpec.Metro
```

Comment:
Can/should we enforce consistency here? For example, if the cluster spec has `metro`, we only look at `metro` for the machine spec?

Reply:
I think we want to leave the flexibility here for folks to manually specify facility overrides while they can. We're already guaranteeing they can't specify both, so that means they've gone out of their way to specify a facility, and we don't want to just ignore that because they specified a metro at the cluster level.